README.md

```diff
@@ -129,6 +129,18 @@ gauge.get({ room: 'kitchen' })
 # => 21.534
 ```
 
+Also you can use gauge as the bi-directional counter:
+
+```ruby
+gauge = Prometheus::Client::Gauge.new(:concurrent_requests_total, '...')
+
+gauge.increment({ service: 'foo' })
```

Comment:
Per the [spec](https://prometheus.io/docs/instrumenting/writing_clientlibs/#gauge) these should be inc and dev, but this also needs redoign to have the right structure.